testsuite/tests/singlecluster/overrides/test_basic_auth.py

```diff
@@ -0,0 +1,42 @@
+"""Test basic enforcement of the rules inside the 'overrides' block of the RateLimitPolicy assigned to a Gateway"""
+
+import pytest
+
+from testsuite.httpx.auth import HttpxOidcClientAuth
+from testsuite.kuadrant.policy.authorization.auth_policy import AuthPolicy
+
+pytestmark = [pytest.mark.kuadrant_only]
+
+
+@pytest.fixture(scope="module")
+def authorization(
+    request, kuadrant, route, gateway, blame, cluster, label, oidc_provider
+):  # pylint: disable=unused-argument
```

Comment:
Tried this, you are right although `route` ended up being necessary there, otherwise the policy wouldn't enforce due to not having a route associated with the gateway created at that point.